<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compiler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="lbl_cpp">
      <property name="text">
       <string>C++</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QTextEdit" name="txt_bin"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lbl_asm">
      <property name="text">
       <string>Assembly</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn_compile">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Compile</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="txt_cpp"/>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="txt_asm"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="lbl_bin">
      <property name="text">
       <string>Binary</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QTableWidget" name="tab_instructions"/>
      </item>
      <item row="2" column="2">
       <widget class="QTableWidget" name="tab_symbols"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_instructions">
        <property name="text">
         <string>Instructions</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbl_symbols">
        <property name="text">
         <string>Symbols</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
